scripts/get_golangci_lint_version.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+grep -oP '^FROM golangci/golangci-lint:v\d+\.\d+\.\d+' "Dockerfile" | grep -oP 'v\d+\.\d+\.\d+'
```

Comment:
just a suggestion, any value in adding some condition that will exit with status 1 incase the version is not found

Reply:
i think ci would fail if this happened anyway